test/bigtest/tests/POLine/line-create-test.js

```diff
@@ -43,17 +43,24 @@ describe('Create PO Line simple test', function () {
 
     describe('Fill values and click save', () => {
       beforeEach(async function () {
+        const purchaseOrder = this.server.schema.orders.first();
+
         await lineEditPage.acquisitionMethod('Approval plan');
         await lineEditPage.selectOrderFormat('Physical resource');
         await lineEditPage.listUnitPrice.fill(3.333);
         await lineEditPage.quantityPhysical.fill(2);
         await lineEditPage.physicalCreateInventory.select('None');
         await lineEditPage.saveButton.click();
-        await orderDetailsPage.whenLoaded();
+
+        const newLine = this.server.schema.lines.first();
+
+        purchaseOrder.update({ ...purchaseOrder.attrs, compositePoLines: [newLine.attrs] });
```

Comment:
what is the reason of  `...purchaseOrder.attrs` ? compositePoLines: [newLine.attrs] should be enough